src/apps/app_muckrock.erl

```diff
@@ -0,0 +1,54 @@
+-module(app_muckrock).
+-export([start/1]).
+-include("ar.hrl").
+
+-define(CHECK_TIME, 60 * 5).
+
+%%% Scrapes and uploads the latest responses received by the Muckrock FOIA
+%%% collection group.
+
+start(WalletFile) ->
+    ssl:start(),
+    Wallet = ar_wallet:load_keyfile(WalletFile),
+    Queue = app_queue:start(Wallet),
+    spawn(fun() -> server(Queue, []) end).
+
+server(Queue, AlreadySeen) ->
+    LatestItems = get_latest(),
+    NewItems = LatestItems -- AlreadySeen,
+    lists:foreach(
+        fun(Item) ->
+            try submit_item(Queue, Item) 
+            catch _:_ -> ok end
+        end,
+        NewItems
+    ),
+    receive
+        stop -> ok
+    after ?CHECK_TIME * 1000 ->
+        server(Queue, AlreadySeen ++ NewItems)
+    end.
+
+submit_item(Queue, Item) ->
+    io:format("Submitting item ~s... ", [Item]),
+    {ok, {{_, 200, _}, _, Body}} = httpc:request(Item),
+    app_queue:add(Queue,
+        #tx {
+            tags =
+                [
+                    {"app_name", "MuckRock"},
+                    {"Original-File-Location", Item},
```

Comment:
I would suggest calling this one `Original-URL` instead.

Reply:
+1